.github/workflows/ci.yml

```
@@ -66,7 +66,15 @@ jobs:
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
       - run: npm ci --legacy-peer-deps
+      - name: Configure Datadog Test Visibility
+        uses: datadog/test-visibility-github-action@v1
```

can use v2 now, they updated the action
```suggestion
        uses: datadog/test-visibility-github-action@v2
```